addons/web/static/tests/views/form/form_renderer.test.js

```diff
@@ -0,0 +1,127 @@
+import { expect, test } from "@odoo/hoot";
```

Comment:
I didn't know about this suite. The question will be: do we merge the 3-4 tests here with form_view.test.js, or do we split form_view.test.js into several sub suites. I know long suites are painful, but there are drawbacks of splitting as well: it's not always clear in which sub suite to put a test, test data (models, views, setup...) are duplicated (and might slightly differ at some point), or centralized in a separated file (not ideal imo, I like having the data on top of the file).